src/config.py

```diff
@@ -55,7 +55,7 @@ class DataStorage(BaseModel):
 
     structural_threshold: int
 
-    temp_dir_path: str
+    temp_dir_path: str = "/tmp"
```

Comment:
I disagree.
From what I know from linux programs there are two types of configuration variables:
* Options that have a default
* Options that don't have a default

Options without a default must be explititly configured by the user. If omitted program startup fails.
Inspiration on possible values can be taken from the example configuration.

Options with a default can be omitted. The default is documented somewhere (mostly in man pages or in the example configuration itself).

My proposal for FACT would be to set the defaults in the pydantic model.
Options that are not set to anything in `main.cfg` will default to whatever is in the model.
The defaults are also documented in the `main.cfg` that we ship.

Edit: I implemented my proposal.

Reply:
Configuring the default values through the main.cfg would have the practical advantage of seeing the default values when configuring FACT without having to dig through the code or docs. Also it would reduce the number of places that you have to look when figuring out where a certain value comes from when debugging. 

Documenting the defaults in comments isn't really a good idea, since nothing forces you to update the comment when changing the default value (which means the comments could get out of date which would be even more confusing). Therefore a comment where to find the values would probably be better or using the auto-generated sphinx docs (the user could be running a different version than the one on Github with different defaults though).

But all in all it doesn't make that much of a difference and if you think this is better then we can do it this way.

Reply:
Either way it is better than it is currently with default values in different places scattered around the code.

Reply:
> and if you think this is better then we can do it this way.

Thank you. I'd like to leave it as it is right now. If it becomes a problem in the future we can get rid of it then without much work.